guides/common/modules/proc_configuring-server-for-use-with-dhcpd.adoc

```diff
@@ -0,0 +1,93 @@
+[id="configuring-server-for-use-with-dhcpd"]
+= Configuring {ProjectServer} for use with dhcpd
+
+You can configure {ProductName} with a non-installer-managed DHCP server.
+
+Perform the steps on the {ProjectServer} or {SmartProxyServer}.
```

Comment:
If this procedure is valid for both Foreman Server _and_ Smart Proxy Servers, I suggest to rename the file name/anchor/title to "Configuring {SmartProxies} for use with dhcpd".

Another suggestion: drop line 5 to combine those two sentences into a single paragraph.

Reply:
Updated as suggested. Thanks.